CHANGELOG.md

```diff
@@ -18,16 +18,21 @@ All user visible changes to this project will be documented in this file. This p
         - Duplicating read users avatars under messages. ([#1243])
         - Invalid message's author being displayed sometimes. ([#1243], [#1050])
         - Gallery image flashing when being opened. ([#1246])
+        - Changing your notes block. ([#1248])
+        - Removing the Actions button from the appbar. ([#1248])
     - Chats tab:
         - Infinite typing indicator occurring sometimes. ([#1243], [#1244])
     - Chat info page:
         - Infinite loader displayed under members list. ([#1246])
+        - Moving info block to the block with photo. ([#1248])
+        - Adding block of monolog chat control. ([#1248])
```

Comment:
These lines are under `Fixed` catagory, yet I don't think those are fixes of some bugs.
Also shouldn't those lines mention this PR as well?